src/test/java/AddCommandTest.java

```diff
@@ -0,0 +1,54 @@
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+class AddCommandTest {
+
+    @Test
+    void shouldPrintInfoOfUserThatHasBeenAddedToUserbook() {
+        User user = new User("test", "test", 1, 1);
+
+        FakeUserbook usersBook = new FakeUserbook(user);
+        FakeConsole console = new FakeConsole("test", 1);
+
+        Command command = new AddCommand(usersBook, console);
+
+        boolean isConsoleEmpty = console.messages.isEmpty();
+        Assertions.assertTrue(isConsoleEmpty);
+
+        command.execute();
+        Assertions.assertTrue(usersBook.userAdded, "method Add is not called");
+
+        boolean isInfoMessageAboutAddingUserIsPresent = console.messages
+                .stream()
+                .anyMatch(message -> message.startsWith
+                        ("New user created: Name = 'test', Surname = 'test', Age = 1', Id = 1"));
+
+        Assertions.assertTrue(isInfoMessageAboutAddingUserIsPresent, "Console-message is Empty");
+        Assertions.assertEquals(user, usersBook.getLastAddedUser());
+
+    }
+
+    @Test
+    void userWillNotBeAddedIfUserEnteredStringInsteadOfIntegerAndExceptionWasThrown (){
```

Comment:
why do you mention about exception in your test case ? You don't check it in test. I think better name for this test case is something like : shouldNotAddUserAndPrintErrorMessageWhenInvalidDataIsEntered()